project/gradle/docker.gradle.kts

```
@@ -0,0 +1,31 @@
+import org.gradle.api.tasks.Exec
+
+/**
+ * Usage:
+ *  - ./gradlew dockerBuild
+ *  - ./gradlew dockerPush
+ */
+
+val dockerImageName: String = (project.findProperty("dockerImageName") as String?) ?: run {
```

detekt (1.23.8)

[warning] Use the safe cast ('as? String') instead of 'as String?'.

(detekt.potential-bugs.CastToNullableType)